.github/workflows/release.yml

```diff
@@ -24,6 +24,8 @@ jobs:
       batch-submitter-service: ${{ steps.packages.outputs.batch-submitter-service }}
       indexer: ${{ steps.packages.outputs.indexer }}
       teleportr: ${{ steps.packages.outputs.teleportr }}
+      go-builder: ${{ steps.packages.outputs.go-builder }}
```

Comment:
What does adding this to the released packages do?

Reply:
It's used by CircleCI to create test builds. This lets us update that like every other Docker image.